.circleci/config.yml

```diff
@@ -63,49 +63,6 @@ workflows:
             tags:
               only: /^v.*/
 
-      - architect/push-to-app-collection:
-          context: architect
-          name: push-chart-operator-to-aws-app-collection
-          app_name: "chart-operator"
-          app_collection_repo: "aws-app-collection"
-          unique: "true"
-          requires:
-            - push-chart-operator-to-aliyun
-            - push-chart-operator-to-control-plane-app-catalog
-          filters:
-            branches:
-              ignore: /.*/
-            tags:
-              only: /^v.*/
-
-      - architect/push-to-app-collection:
-          context: architect
-          name: push-chart-operator-to-azure-app-collection
-          app_name: "chart-operator"
-          app_collection_repo: "azure-app-collection"
-          unique: "true"
-          requires:
-            - push-chart-operator-to-control-plane-app-catalog
-          filters:
-            branches:
-              ignore: /.*/
-            tags:
-              only: /^v.*/
-
-      - architect/push-to-app-collection:
-          context: architect
-          name: push-chart-operator-to-kvm-app-collection
-          app_name: "chart-operator"
-          app_collection_repo: "kvm-app-collection"
-          unique: "true"
-          requires:
-            - push-chart-operator-to-control-plane-app-catalog
-          filters:
-            branches:
-              ignore: /.*/
-            tags:
-              only: /^v.*/
-
```

Comment:
Disabling the push to app collections due to customer change freeze. Will re-enable on Monday for the next release.

Reply:
What do you think we introduce a button to trigger releasing to the app-collection? It's more flexible than what we used to do.

Reply:
That's a cool idea. We could add a hold step like for the e2e tests in azure-operator.

But usually we just want to release to the CPs and I'm going to add this back on Monday. We can do this if we need to disable the push to collections more often.